src/main/webapp/dev/js/main/instructorCourseEdit.js

```diff
@@ -38,6 +38,9 @@ const instructorPrivilegeValues = [
     'canmodifysessioncommentinsection',
 ];
 
+const WARNING_DISCARD_CHANGES = 'Warning: Any unsaved changes will be lost';
```

Comment:
Let's introduce this constant in the `Const.js` file

Reply:
I realized `instructorFeedbackEdit.js` is also declaring localized constants inside the file itself and not using values from `Const.js`, whose constants seem more general. 

How should I name these constants if I were to put it in `Const.js`?

Reply:
I went through multiple JS files and these `Const` are already everywhere. I guess it's alright to leave it the way it is rather than introduce these messages in `const.js`.